src/components/starlight/SiteTitle.astro

```diff
@@ -48,14 +49,36 @@ const href = pathWithBase((Astro.props.locale || '') + '/getting-started/');
 		text-decoration: none;
 		white-space: nowrap;
 	}
+	.site-title:focus-within {
+		overflow: visible;
+	}
 	.site-title a:first-child {
 		padding-top: 0.8rem;
 	}
 	.site-title a {
-		transition: transform 0.18s ease-out;
+		position: relative;
+		margin-top: calc(var(--hover-offset) * -1);
+		margin-bottom: calc(var(--hover-offset) * -1);
+		transition: transform 0.18s cubic-bezier(0.23, 1, 0.32, 1);
 	}
 	.site-title a:hover {
-		transform: translateY(-0.25rem);
+		transform: translateY(calc(var(--hover-offset) * -1));
+	}
+	.site-title a:focus-visible {
+		outline: 0;
+	}
+	.site-title a:focus-visible::after {
+		content: '';
+		position: absolute;
+		top: var(--sl-nav-pad-y);
+		right: 0;
+		bottom: var(--sl-nav-pad-y);
+		left: 0;
+		outline: var(--sl-color-text-accent) auto 2px;
+	}
+	.site-title a:first-child:focus-visible::after {
+		left: -0.5em;
+		right: 0.25em;
 	}
```

Comment:
Might be a cheaper fix if we just `outline-offset: -3px` or something on the `a` directly? That will move the outline inside so it’s visible.

Reply:
That would be cheaper, yes! I started there but quickly found it much trickier than I anticipated. The height adjustments required for the `:hover` behavior made changing the `outline-offset` impossible. `outline-offset` is applied consistently in all four directions, but we have to bring in the block axis while maintaining the inline axis.

Since we needed more control than the `offset` properties provided, the pseudo-element made the most sense.

Reply:
Hmm now that I think of it, it might be easier to apply the outline to the `svg` element rather than the `a` element. Let me try that quickly.